### ISSUE TITLE
Abnormal production of facial expressions

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7321]
**Preferred term label:**
Abnormal production of facial expressions

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormality in the subdomain of nonverbal communication which involves a reduction in or atypical production of emotional facial expressions.
PMID: 26803417 (This article includes a list with defs of potential child terms.)

**Parent term (use hpo.jax.org/app)**
Impaired use of nonverbal behaviors
HP:0000758

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by bcoleman96** [comment]
Might want to create a parallel term for abnormal interpretation of facial expressions.

**Comment by pnrobinson** [comment]
added as HP:4000074
